source/code/troubleshooter/omsagent_tsg_zip/tsg_files/tsg_code/connect/tsg_checke2e.py

```diff
@@ -0,0 +1,71 @@
+# INSPIRED BY verify_e2e.py
+
+import subprocess
+
+from tsg_errors import tsg_error_info, get_input
+
+def check_e2e():
+    # get machine's hostname
+    hostname = subprocess.check_output(['hostname'], universal_newlines=True).rstrip('\n')
+
+    sources = ['Heartbeat', 'Syslog', 'Perf']
+
+    successes = []
+    failures = []
+
+    print("--------------------------------------------------------------------------------")
+    print(" Please go to https://ms.portal.azure.com and navigate to your workspace.\n"\
```

Comment:
I believe for it is just `portal.azure.com`, `ms.` is just for internal Microsoft

Reply:
Didn't realize that, updated the link accordingly:
https://github.com/microsoft/OMS-Agent-for-Linux/blob/bff7c80f869d6f5a4046ed41b4bd0ed89d69219f/source/code/troubleshooter/modules/connect/check_e2e.py#L18